### DISCUSSION TITLE
Welcome to microsoft-github-training Discussions!

### DISCUSSION THREAD

**Discussion by AKThu** [AKThu/microsoft-github-training#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AKThu** [comment]
Hi. I'm the owner of this repository and I do whatever I want

**Reply by AKThu002** [reply]
> Hi. I'm the owner of this repository and I do whatever I want

Are you serious bro

**Reply by AKThu** [reply]
> > Hi. I'm the owner of this repository and I do whatever I want
> 
> Are you serious bro

yes

**Comment by AKThu002** [comment]
Hello. I'm the 2nd member of this repository and I don't even know why am I here.

**Reply by AKThu** [reply]
nuts
